tests/general/LineContinuationBearTest.py

```diff
@@ -0,0 +1,20 @@
+from bears.general.LineContinuationBear import LineContinuationBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+a = some_function(
+    '1' + '2' + '3' - '4')
+"""
+
+bad_file = r"""
+a = '1' \
+    + '2' \
+    + '3'
+"""
+
+LineContinuationBearPythonTest = verify_local_bear(LineContinuationBear,
+                                                   valid_files=(good_file,),
+                                                   invalid_files=(bad_file,),
+                                                   settings={'language':
+                                                             'Python'})
```

Comment:
Can you add `self.assertEqual` methods (e.g. KeywordBearTest) to verify the behavior of your bear?

Reply:
Hmm, can you give me an example?

Reply:
For example, how do we know the line number and column number you generate is correct? You have sth like `+1`, etc., which may or may not be true. Adding tests help reviewers check your bear has the correct behavior.